src/main/java/teammates/ui/template/FeedbackSessionPublishButton.java

```diff
@@ -25,6 +25,8 @@ public FeedbackSessionPublishButton(PageData data, FeedbackSessionAttributes ses
         this.feedbackSessionName = Sanitizer.sanitizeForJs(session.feedbackSessionName);
         this.isSendingPublishedEmail = session.isPublishedEmailEnabled;
 
+        String returnUrl = isHome ? Const.ActionURIs.INSTRUCTOR_HOME_PAGE 
+                                  : Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE;
```

Comment:
Is there a reason for passing in `isHome` as a parameter rather than `returnUrl`?